shelf/bucket_update/search_updater.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, container):
         """
         self.container = container
         self.bucket_container = self.container.bucket_container
-        self.chunk_size = self.container.config["chunkSize"]
+        self.chunk_size = self.container.config.get("chunkSize")
```

Comment:
This is where it starts to get strange. In one of our actions this is absolutely required. In one of our other actions it is not used at all.

I wont ask for you to change it at this point though